addons/website/static/tests/tour_utils/widget_lifecycle_dep_widget.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import publicWidget from "web.public.widget";
+
+window.localStorage.WidgetAndWysiwygLifecycle = window.localStorage.WidgetAndWysiwygLifecycle || "[]";
+
+publicWidget.registry["countdown_patch"] = publicWidget.Widget.extend({
```

Comment:
Detail as it is in a test but we now should name public widget registry entries as classes: so here: `registry.CountdownPatch`